tools/doc/buildCSSForFlavoredJS.mjs

```diff
@@ -0,0 +1,11 @@
+export default function buildCSSForFlavoredJS(dynamicSizes) {
+  if (dynamicSizes == null || dynamicSizes.length === 0) return '';
+
+  return `<style>${Array.from(dynamicSizes, (charCount) =>
+    `@media(max-width:${charCount * 8 + 142 + 16 * 4 + (charCount > 68 ? 274 : 0)}px){` +
+      `.with-${charCount}-chars>.js-flavor-selector{` +
+        'float:none;' +
+        'margin:0 0 1em auto;' +
```

Comment:
Since it's right-aligned now, we don't have to reset top and right margins. :)
```suggestion
        'margin-bottom:1em;' +
        'margin-left:auto;' +
```

Reply:
`'margin:0 0 1em auto;'.length <  ('margin-bottom:1em;' + 'margin-left:auto;').length`, I think we should keep the current form (keep in mind this string is repeated in the HTML files once[^1] for each snippet block)

[^1]: except when the first two lines of several blocks have the same length, e.g. in `all.html` there are 376 dual-mode JS snippets, and only 52 distinct media queries injections.